omod/src/main/webapp/resources/scripts/order-entry.js

```diff
@@ -229,14 +229,16 @@ angular.module("orderEntry", ['orderService', 'encounterService', 'session'])
                     delete transformed.editing;
                     return transformed;
                 });
+                
                 var encounter = {
                     patient: encounterContext.patient.uuid,
                     encounterType: encounterContext.encounterType.uuid,
                     visit: uuidIfNotNull(encounterContext.visit),
                     location: uuidIfNotNull(encounterContext.location),
-                    provider: provider.person.uuid, // submit the person because of RESTWS-443
+                    // provider: provider.person.uuid, // submit the person because of RESTWS-443
```

Comment:
Specifically there's an `encounterProviders` property which replaced the
old `provider` property in openmrs-core 1.9.